modules/programs/git.nix

```diff
@@ -299,5 +315,13 @@ in {
             ([ "git-lfs" "smudge" ] ++ skipArg ++ [ "--" "%f" ]);
         };
     })
+
+    (mkIf cfg.delta.enable {
+      home.packages = [ pkgs.gitAndTools.delta ];
+
+      programs.git.iniContent.core = {
+        pager = concatStringsSep " " ([ "delta" ] ++ cfg.delta.options);
```

Comment:
Wouldn't it be a bit cleaner to point directly into the nix store here (i.e. `[ pkgs.gitAndTools.delta ]`) instead extending `home.packages`? I assume that most users do not need `delta` in their `PATH`, perhaps in contrast to `git lfs` configured in the same way.

Reply:
fair enough, I've removed delta from `home.packages`